bedrock/settings/base.py

```diff
@@ -252,6 +252,8 @@ def lazy_langs():
     # remote FTL files from l10n team
     FLUENT_REPO_PATH,
 ]
+FLUENT_MIGRATIONS = 'lib.fluent_migrations'
+FLUENT_MIGRATIONS_PATH = ROOT_PATH / 'lib' / 'fluent_migrations'
```

Comment:
can these live in the `FLUENT_LOCAL_PATH`, or should they stay with the code? It seems like we'll want/need to delete them once the migration makes it to production.

Reply:
One thing that the migration library does is that it loads them as python modules.

I didn't feel great about that being in a data path, so I moved them to `lib`. Yes, I started with them being in `l10n` ;-) .

In Firefox, they're actually just part of the default virtualenv path, too.

And yes, they should stay with the code. I see how that's a tad tricky to ensure that we're doing things right, but there's also not a whole lot of risk. One thing the migration library does is that if a string is already in the target, it won't overwrite it.

I wouldn't recommend to overwrite all strings with all recipes all the time, but it's kinda safe and won't destroy any updates that localizers did, for example.

And yes, we clean those recipes up on the Firefox side once we don't need them around anymore. On bedrock, this could probably be an issue to do once the migrated content made the roundtrip through the repos to prod. Or as we do on fx, just a kinda-monthly chore.